config/googlechatconf.go

```diff
@@ -0,0 +1,29 @@
+package config
+
+import (
+	"github.com/asaskevich/govalidator"
+	"golang.org/x/xerrors"
+)
+
+// GoogleChatConf is GoogleChat config
+type GoogleChatConf struct {
+	WebHookURL  string `json:"-" toml:"webHookURL,omitempty"`
+	SkipHealthy bool   `json:"-" toml:"skipHealthy"`
+	Filter      string `json:"-" toml:"filter,omitempty"`
+	Enabled     bool   `json:"-" toml:"-"`
+}
+
+// Validate validates configuration
+func (c *GoogleChatConf) Validate() (errs []error) {
+	if !c.Enabled {
+		return
+	}
+	if len(c.WebHookURL) == 0 {
+		errs = append(errs, xerrors.New("googleChatConf.webHookURL must not be empty"))
+	}
+	_, err := govalidator.ValidateStruct(c)
```

Comment:
Stop calling ValidateStruct because the struct tag does not have a validation rule .

Reply:
I understood about govalidator. I rewrite.